src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -29,14 +29,71 @@ def ges_pflegev_zusatz_kinderlos(
     return out
 
 
+def ges_pflegev_beitr_regulär_besch_m(
```

Comment:
We should use two functions for the situation before 2023-07 and the situation after and make use of the `@dates_active` decorator (see the code base for several examples). 

Then, we don't need to define the parameter `abschlag_kinder` for the time before that.